File: src/test/testData/inspections/ViewReferenceLeak.java
import android.app.Fragment;
import android.view.View;

public class ViewReferenceLeak extends Fragment {
    // Bad: Not nulled in onDestroyView
    private View <error descr="LeakLens: View field 'myView' is not nulled in onDestroyView().">myView</error>;

    // Good: Nulled in onDestroyView
    private View mySafeView;

    public void onDestroyView() {
        super.onDestroyView();
        mySafeView = null;
    }
}

File: src/test/testData/inspections/ViewReferenceLeak.kt
import android.app.Fragment
import android.view.View

class ViewReferenceLeakKt : Fragment() {
    // Bad: Not nulled in onDestroyView
    private var <error descr =
        "LeakLens: View field 'myView' is not nulled in onDestroyView()." > myView < / error >: View? = null

    // Good: Nulled in onDestroyView
    private var mySafeView: View? = null

    override fun onDestroyView() {
        super.onDestroyView()
        mySafeView = null
    }
}
